core/src/createQR.ts

```diff
@@ -22,8 +23,7 @@ export function createQR(url: string | URL, size = 512, background = 'white', co
     return new QRCodeStyling(createQROptions(url, size, background, color));
 }
 
-/** @ignore */
-export function createQROptions(url: string | URL, size = 512, background = 'white', color = 'black'): Options {
+function createQROptions(url: string | URL, size = 512, background = 'white', color = 'black', gradient?: Gradient): Options {
```

Comment:
Thanks for the PR! Can you keep this function exported please, we don't want to break existing callers

Reply:
It's an optional parameter, so existing callers won't see any differences.
I tested it prior commiting and it works as before without the gradient param.